addons/website/static/src/builder/translation_components/selectTranslateDialog.js

```diff
@@ -12,20 +12,26 @@ export class SelectTranslateDialog extends Component {
         close: Function,
     };
     setup() {
-        this.inputEl = useRef("input");
+        this.inputEl = useRef("translateInputs");
```

Comment:
```suggestion
        this.inputEls = useRef("translateInputs");
```
or maybe translateEl as it is div?